app/og/route.tsx

```diff
@@ -8,7 +7,7 @@ const width = 1200;
 export async function GET(request: Request) {
   try {
     const { searchParams } = new URL(request.url);
-
+    const origin = `${request.headers.get("x-forwarded-proto") || "http"}://${request.headers.get("host")}`;
```

Comment:
Just before I merge. What exactly does this do? 🤔

Reply:
In an open graph image we can't use relative path directly in the src attribute.
So I had to use the actual path to the image.
Example : http://localhost:3000/images/og/<image-name> 
Works when used in the src attribute.

The code takes the request origin and creates the absolute path to the images.